VOKMockUrlProtocol.m

```diff
@@ -155,6 +166,10 @@ - (NSArray *)resourceNames
                 [resourceNames addObject:[[name stringByAppendingFormat:AppendSeparatorFormat, bodyString] mutableCopy]];
             }
             [resourceNames addObject:[[name stringByAppendingFormat:AppendSeparatorFormat, bodyHash] mutableCopy]];
+            
+            if (AllowsWildcard) {
+                [resourceNames addObject:[[name stringByAppendingFormat:AppendSeparatorFormat, Wildcard] mutableCopy]];
+            }
         }
     }
```

Comment:
Yes, that's the ordering you'd get for a request with a query and a body.  If you'd like, I can change it to force all the wildcard variations to the end.